content/accessibility/tools.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Accessibility tools
+---
+
+A selection of tools to help product designers design accessibly. 
+
+## Figma 
+
+### [Accessibility sticker sheet](https://www.figma.com/file/o3Vhxw4SmBsUjeaVq3nF4y/Accessibility-Notation?node-id=0%3A1)
+This Figma file includes annotations to use on your designs prior to developer hand off. These annotations let you note accessibility information like desired tab order, screen reader labels, headings, and landmarks. Pull directly from the file or turn on the accessibility annotations in your Figma assets list.
+
+![""](https://user-images.githubusercontent.com/40274682/89364293-04f02f00-d687-11ea-9542-439fa61ef3c8.png)
```

Comment:
Since the other screenshots on this page don't have a border around, should we remove it from here, or does it become hard to see?

Reply:
The guidelines for a screenshot on primer docs say to use the grey-100 background (https://www.figma.com/file/YQT3nlmvSdZRPBnkDXvTFk/Doctocat-image-template?node-id=0%3A1). I could get a different screenshot that fills the area better (it also seems low quality and I want to fix that)